leits/MeetingBar
# Fixed Polish translation

### Status
**READY**

### Description
Current Polish translation mistranslates some of the words but also contains weirdly quoted parts of the strings file that break the UI.


## Checklist

### Steps to Test or Reproduce
Open preferences and see for yourself:

<img width="690" alt="image" src="https://user-images.githubusercontent.com/81205/135060942-be5f64ff-9a8a-4fbf-a1df-bb3ced8f99a1.png">
